holoviews/core/data/interface.py

```diff
@@ -162,6 +166,8 @@ def initialize(cls, eltype, data, kdims, vdims, datatype=None):
             try:
                 (data, dims, extra_kws) = interface.init(eltype, data, kdims, vdims)
                 break
+            except DataError:
+                raise
```

Comment:
Isn't something similar needed in the try/except where all the data interfaces are tried, one by one?

Reply:
I'm confused, that's exactly where this is.